### DISCUSSION TITLE
Failed to build misaka - ISSO installation failed ... Debian

### DISCUSSION THREAD

**Discussion by jalsco** [posativ/isso#794] [Q&A] [answered]
I just trying to install onto: `Debian 10 (buster), Python 3.7.3`
Installation fails : 

` Building wheel for misaka (setup.py) ... error
  ERROR: Command errored out with exit status 1:'

**Full errors attached......**    

![image](https://user-images.githubusercontent.com/7985459/143266567-e03d87ad-bc4a-48c3-845f-46aec778fd20.png)

**Comment by ix5** [comment, marked as answer]
Hello @jalsco, it'd be much more convenient for other people trying to help you if you pasted the error you received in text form instead of a hardly readable screenshot. Not everybody is able to decipher dark red text on a dark background, and some people rely on screen readers or other accessibility tools.
To paste blocks of code, you surround the block in triple backticks (```)

---

First, please state the exact steps you took while installing. Which debian packages did you install before attempting to run the `pip` command that errors out for you?

Second, installing isso directly via `pip` from the `master` branch is not one of the documented options.

Third, since you helpfully pasted the log of the error you received, here's the important part:
```
[...]
fatal error: pyconfig.h no such file or directory
[...]
compilation terminated
```

[Researching the issue](https://lmgtfy.app/?q=pyconfig.h+no+such+file+or+directory) surfaces a few solutions which you might try, for example [this one](https://github.com/stepjam/PyRep/issues/62).

The [isso documentation for installation](https://posativ.org/isso/docs/install/#install-from-pypi) instructs you to install following system packages before attempting to install `isso` via `pip`:
```
sudo apt-get install python-dev sqlite3 build-essential
```

For python3, that should be `python3-dev` (since Debian sadly still seems to default to python2). So your issue seems to be that you do not have the correct python development package installed.

If you'd like, you can open a Pull Request to change that in the documentation. Isso does not support python2 since https://github.com/posativ/isso/pull/613.

**Comment by jalsco** [comment]
Thank you for your reply. I will try those steps and report back. 
Also - since posting that  I have upgraded to Debian 11 (bullseye), Python 3.9.

**Comment by ix5** [comment]
@jalsco did you ever get to test out the suggestions presented to you?

If so, would you mind creating that PR to the documentation to help out others with the same issue?
